app/views/track.js

```diff
@@ -78,12 +86,22 @@ const TrackView = Mn.LayoutView.extend({
 
     select(e) {
         e.preventDefault();
+
         let selectedTracks = application.selected.get('tracks');
+        let isSelected = selectedTracks.get(this.model.get('_id'));
+        let isMenuTouch = $(e.target).parents('#popup-menu').length;
+        let width = $(window).width();
+
+        // If we are on mobile play the track on touch
+        if (width < 1024 && isSelected && !isMenuTouch) {
```

Comment:
I'm not sure rely on width to _detect_ if the device is a mobile or not a good option. Prefer use a TouchEvents or PointerEvents capabilities detection to enable touch/mobile features (probably a safer way than rely on an arbitrary variable like width). You can take a look at Modernizr tests.